lib/matplotlib/sankey.py

```diff
@@ -504,28 +512,35 @@ def add(self, patchlabel='', flows=None, orientations=None, labels='',
                 "cause poor layout.\nConsider changing the scale so"
                 " that the scaled sum is approximately 1.0." % gain, 'helpful')
         if prior is not None:
-            assert prior >= 0, "The index of the prior diagram is negative."
-            assert min(connect) >= 0, (
+            if prior < 0:
+                raise ValueError("The index of the prior diagram is negative.")
+            if min(connect) < 0:
+                raise ValueError(
                 "At least one of the connection indices is negative.")
-            assert prior < len(self.diagrams), (
+            if prior >= len(self.diagrams):
+                raise ValueError(
                 "The index of the prior diagram is %d, but there are "
                 "only %d other diagrams.\nThe index is zero-based."
                 % (prior, len(self.diagrams)))
-            assert connect[0] < len(self.diagrams[prior].flows), (
+            if connect[0] >= len(self.diagrams[prior].flows):
+                raise ValueError(
                 "The connection index to the source diagram is %d, but "
                 "that diagram has only %d flows.\nThe index is zero-based."
                 % (connect[0], len(self.diagrams[prior].flows)))
-            assert connect[1] < n, (
+            if connect[1] >= n:
+                raise ValueError(
                 "The connection index to this diagram is %d, but this diagram"
                 "has only %d flows.\n The index is zero-based."
                 % (connect[1], n))
-            assert self.diagrams[prior].angles[connect[0]] is not None, (
```

Comment:
probably in a previous commit I had ``if ... is not None``, because I forgot to remove the ``not`` when removing the assert